linter/src/firebase/fetching.cpp

```diff
@@ -212,17 +189,13 @@ firebase_request(
         }
 
         res = curl_easy_perform(curl);
-        if (res != CURLE_OK) {
-            log_e(firebase, "curl_easy_perform() failed: {}", curl_easy_strerror(res));
-        }
 
         curl_easy_cleanup(curl);
     }
     glz::json_t json{};
     auto error = glz::read_json(json, readBuffer);
```

Comment:
yyou should probably check `res` here on L222 before bothering to read the JSON.